documentation/source/newsfragments/2079.bugfix.rst

```diff
@@ -0,0 +1,8 @@
+Generate blocks are now accessible directly via lookup without having to iterate over parent handle. (:pr:`2079`)
+
+  .. code-block:: python3
+
+      # Example pseudo-region
+      dut.genblk1       #<class 'cocotb.handle.HierarchyArrayObject'>
+
+  .. consume the towncrier issue number on this line.
```

Comment:
Is this line a pragma? Is it doing anything?

Reply:
Towncrier will always append the issue/PR number to the last non-empty line of a newsfragment, and it doesn't respect ReST markup or anything.

The comment is a small hack to neutralize this append step.